web/app/js/components/util/TapUtils.jsx

```diff
@@ -163,52 +163,88 @@ const publicAddressToString = ipv4 => {
 */
 const resourceShortLink = (resourceType, labels, ResourceLink) => (
   <ResourceLink
-    key={labels[resourceType]}
+    key={`${labels[resourceType]}-${labels.namespace}`}
     resource={{ type: resourceType, name: labels[resourceType], namespace: labels.namespace}}
     linkText={toShortResourceName(resourceType) + "/" + labels[resourceType]} />
 );
 
-// display consists of a list of ips and pods for aggregated displays (Top)
-const displayLimit = 3;
-const resourceSection = (endpoint, display, labels, ResourceLink) => {
-  let ipList = null;
-  let podList = null;
+const displayLimit = 3; // how many upstreams/downstreams to display in the popover table
+const popoverSrcDstColumns = [
+  { title: "Source", dataIndex: "source", key: "source" },
+  { title: "Destination", dataIndex: "destination", key: "destination" }
```

Comment:
Would it be possible to add another column between Source and Destination, with some sort of icon that indicates the direction of the traffic? It's a bit hard to parse visually that traffic is flowing from column 1 to column 2. For instance:

![image](https://user-images.githubusercontent.com/9226/45985673-2b20fe80-c01c-11e8-9752-5e1da9d35ca8.png)

In this example it's hard to tell that there are two columns. I think maybe adding something like `10.1.01 ➡️ 10.1.14.247` would make it clear.

Reply:
ooooh, interesting. sure!

Reply:
![screen shot 2018-09-24 at 5 17 55 pm](https://user-images.githubusercontent.com/549258/45986011-db433700-c01d-11e8-9af6-c68223d2bc94.png)

Reply:
Thanks! At the risk of bike-shedding this wayyy too much, how about:

```
  { title: "", render: () => <i className="fas fa-long-arrow-alt-right" />},
```

Reply:
haha sure!